compiler/crates/relay-lsp/src/goto_definition/mod.rs

```diff
@@ -44,6 +44,15 @@ pub enum DefinitionDescription {
         parent_type: Type,
         field_name: StringKey,
     },
+    FieldArgument {
+        parent_type: Type,
+        field_name: StringKey,
+        argument_name: StringKey,
+    },
+    DirectiveArgument {
+        directive_name: StringKey,
```

Comment:
Don't we have a [new type](https://doc.rust-lang.org/rust-by-example/generics/new_types.html) for directive name, and perhaps argument name? If so, could we use them here?